functional.py

```diff
@@ -0,0 +1,62 @@
+import sys
+
+def sequential_map(*args):
+    object = args[-1]
+    if type(object) == int:
+        for func in args[:-1]:
+            object = func(object)
+    else:
+        for func in args[:-1]:
+            object = list(map(func, object))
+    return object
+
+
+def func_chain(*args):
+    def result(object):
+        if type(object)==int:  #for numbers
+            for func in args:
+                object = func(object)
+        else:  #for lists and tuples
+            for func in args:
+                object = list(map(func, object))
+        return object
+    return result
+
+
+def consensus_filter(*args):
+    object = args[-1]
+    if type(object) == int:  # for integer objects
+        if args[0](object):
+            return object
+        else:
+            return None
+    for func in args[:-1]:  # for sets and lists
+        right_nums = list(map(func, object))  # list of True and False
```

Comment:
Опять же, из map'а не надо делать список, если ты собираешься по нему итерироваться, это лишние вычислительные операции
```suggestion
        right_nums = map(func, object)  # list of True and False
```